### ISSUE TITLE
W3C ODRL Recommendations errata maintenance

### ISSUE THREAD

**Issue by riannella** [w3c/odrl#6]
Errata Issue List: https://github.com/w3c/poe/issues
Errata Process: https://www.w3.org/2016/poe/errata/

**Comment by vroddon** [comment]
We have still 7 open issues. I see the errata are still there...

**Comment by riannella** [comment]
Yes, step 1 is to confirm each actually is an Errata and set the label as such.

Propose here the one that are/are not...

**Comment by vroddon** [comment]
As per the guidelines at https://www.w3.org/2016/poe/errata/, 

> The community discusses the issue. If it is accepted as a genuine erratum, the label “Errata” is added to the entry

I suggest that in the following call, each ErratumRaised is reviewed one by one, and upon agreement, the tag "Errata" is added. If not enough quorum is reached, then we suggest a cross-review policy, avoiding that one member can tag as Errata an issue raised by the same member.

**Comment by riannella** [comment]
See https://github.com/w3c/odrl/pull/33
